pages/blog/status-update-41-20.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "AsyncAPI Initiative Status Update (week 41, 2020)"
+date: 2020-10-06T06:00:00+01:00
+type: Communication
+featured: true
+tags:
+  - Status
+  - Community
+cover: /img/posts/status-update-41-20-new-website.webp
+authors:
+  - name: Lukasz Gornicki
+    photo: /img/avatars/lpgornicki.webp
+    link: https://twitter.com/derberq
+    byline: AsyncAPI Maintainer and Community Guardian
+---
+
+
+
+## New Website
+
+I'm super excited to share with you that last week we released [a new AsyncAPI website](/):
+- On a home page, we now promote [the Playground](https://playground.asyncapi.io) more and put much more emphasis on the different ways you can engage with AsyncAPI Community,
+- Documentation view got additional navigation per document so, for example, it is now much better to navigate through different sections of [the specification document](/docs/specifications/2.0.0),
+- The tools section is now dedicated to AsyncAPI's official tools to clarify the AsyncAPI initiative is not just the spec. Just have a look how much detailed is, for example, the [generator view](/generator),
+- Don't worry, all the other tools are still there, under [the Community section](/docs/community/tooling)
+- The cherry on the cake is a much more appealing [Blog view](/blog)
+
+[Fran Méndez](https://twitter.com/fmvilas) did all this fantastic job. Well done, Bro! Sources of the website are here. Feel free to use it as a template for your website and let us know if you need help reusing it.
+
+## Hacktoberfest
+
+AsyncAPI Initiative joined Hacktoberfest. We decided to join the event at the end of August when nobody even thought this would happen. The first day of Hacktoberfest felt like a false start. Because of the enormous amount of SPAM pull requests, some open-source community people went too far in the #cancelculture trend. They opted on Twitter for the Hacktoberfest cancelation and DigitalOcean boycott. Luckily not everyone is haters and Hacktoberfest stays. Anyway, this is a topic for another post.
```

Comment:
```suggestion
AsyncAPI Initiative joined Hacktoberfest. We decided to join the event at the end of August when nobody even thought this would happen. The first day of Hacktoberfest felt like a false start. Because of the enormous amount of spam pull requests, some open-source community people went too far in the #cancelculture trend. They opted on Twitter for the Hacktoberfest cancelation and DigitalOcean boycott. Luckily, not everyone is a hater, and Hacktoberfest stays. Anyway, this is a topic for another post.
```

* SPAM (all caps) looks like it's an acronym.